BoxContentSDK/BoxContentSDK/Requests/BOXFileCommentsRequest.m

```diff
@@ -38,7 +38,7 @@ - (BOXAPIOperation *)createOperation
     NSMutableDictionary *parameters = [NSMutableDictionary dictionary];
     
     if (self.requestAllItemFields) {
-        parameters[BOXAPIParameterKeyFields] = [self fullCommentFieldsParameterString];
+        parameters[BOXAPIParameterKeyFields] = [self fullItemFieldsParameterStringExcludingFields:self.fieldsToExclude];
```

Comment:
This doesn't seem right, before it was fullCommentFields, now it's itemFields. I don't think that's what we want